llvm/test/CodeGen/Generic/atomic-scalarization.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc %s --print-after-isel --disable-verify 2>&1 | FileCheck %s
```

Comment:
Don't use print-after-isel, or -disable-verify. Check the final isa output (or -stop-after to check real mir), and move to test/CodeGen/X86

Reply:
Maybe -stop-after=finalize-isel

Reply:
Why `-stop-after=finalize-isel` yet not `--print-after-isel`?